sdv/multi_table/base.py

```diff
@@ -473,3 +474,25 @@ def get_info(self):
             'last_fit_date': self._fitted_date,
             'fitted_sdv_version': self._fitted_sdv_version
         }
+
+    def save(self, filepath):
+        """Save this instance to the given path using cloudpickle.
+
+        Args:
+            filepath (str):
+                Path where the instance will be serialized.
+        """
+        with open(filepath, 'wb') as output:
+            cloudpickle.dump(self, output)
+
+    @classmethod
+    def load(cls, path):
+        """Load the multi-table synthesizer from a given path.
+
+        Args:
+            path (str):
+                Path from which to load the instance.
+        """
+        with open(path, 'rb') as f:
+            synthesizer = cloudpickle.load(f)
+            return synthesizer
```

Comment:
I think that we can get inspired by the `single_table` and use the  `throw_version_mismatch_warning(getattr(synthesizer, '_package_versions', None))`